src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx

```diff
@@ -0,0 +1,214 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Default value
+      </td>
+
+      <td>
+        Optional. This is the default value that the dashboard will filter on. For example, if you used the `country` query above, you could input `ES` as the default value and the dashboard would automatically filter to that value. 
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+For an example of what a template variable for `country` values would look like, see [the screenshot at the top of this doc](#defined). 
+
+Once you've defined your template variable, you can move on to adding a template variable to your widgets. 
+
+### Step 2. Create widgets that use the variable [#create-widgets]
+
+Once you've configured a template variable, you'll need widgets on a dashboard that use the variable that you've defined in their query.
+
+Here's an example of creating a query that uses the `country` variable we mentioned in part 1: 
+
+```
+SELECT countryCode FROM PageAction WHERE countryCode = {{countryCode}}
+```
+
+A couple important points to note about using a variable in a query: 
+
+* The variable you defined goes inside the `{{ … }}` brackets. 
+* The variable generates a string value.  
+
+And here's an example of the resulting widget, on the right, with the `country` dropdown to the left. 
+
+<img
+  title="Country template variable example"
+  alt="Country template variable example"
+  src={dashboardsCountryVariableExample}
+/>
+
+When you're done a) defining a template variable and b) adding a widget that references that variable, you can verify it's working as expected by choosing different options from the template variable bar and seeing if the widget changes based on your selection. 
+
+## Rules for writing a query-type template variable [#query-variable-rules]
+
+As discussed in the [section on defining template variables](#define-variable), there are three variable types: query, list, and text field. The query-type variable is the most complex to create because you must create a working query that returns values, which are then used to populate the dropdown in the template variable bar at the top of the dashboard. 
+
+<Callout variant="important">
+Note that this is a different topic than writing queries that make use of a template variable.
+</Callout>
+
+Here are some rules and guidance on creating a query-type variable: 
+* Queries must use the `unique` syntax. For example: `From PageAction select uniques(countryCode)`
+* You can use almost any NRQL query as long as the `uniques` component returns a list of values.
+* Nested variables are not supported: there can't be variables within variables.
+
+## Examples [#examples]
+
+Here are some different types of template variable implementations. 
+
+<CollapserGroup>
+  <Collapser
+    id="multiple-variables"
+    title="Using multiple variables for one widget"
+  >
+
+You can define one template variable at a time, but you can use more than one template variable in a single widget. 
```

Comment:
This reads a little weirdly to me, I think I'm expecting it to be "but you can't" since it started with "you can". What do you think about replacing "but you can" with "and you can also"?

Reply:
lol. yeah that's a bad one. whoops.